core/src/main/java/org/elasticsearch/ingest/PipelineStore.java

```diff
@@ -51,6 +51,7 @@
 
     private final Pipeline.Factory factory = new Pipeline.Factory();
     private final Map<String, Processor.Factory> processorFactories;
+    private final boolean newDateFormat;
```

Comment:
Maybe rename to `newIngestTimestampFormat`, since it only applies to ingest timestamp?